scripts/observers/stop.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+os.system("killall screen")
```

Comment:
Will kill all `screen` instances that the user has. Perhaps we should find another solution for this, or remove the `stop.py` script in order to avoid undesired effects?

Reply:
changed